tests/test_tydiqa_secondary.py

```diff
@@ -15,3 +18,18 @@ def test_prompt():
         "such as wound, ostomy, and continence nursing and burn center care.\n"
     ) in prompt
     assert prompt.endswith("Answer:")
+
+
+def test_promptsource_template():
+    ds_key, sub_key = "tydiqa", "secondary_task"
+    tydiqa_sec_vld_ds = load_dataset(ds_key, sub_key, split="validation", streaming=True)
+    tydiqa_sec_vld_ds_en = filter(lambda x: x["id"].split("-")[0] == "english", tydiqa_sec_vld_ds)
+    template_collection = TemplateCollection()
+    tydiqa_sec_tmpls = template_collection.get_dataset(ds_key, sub_key)
+    tmpl = tydiqa_sec_tmpls["simple_question_reading_comp_2"]
+    prompt, _ = tmpl.apply(removeHyphen(next(tydiqa_sec_vld_ds_en)))
```

Comment:
Although only doing `removeHyphen()` here, `promptsource` has some more preprocessing for classification, see https://github.com/bigscience-workshop/promptsource/blob/main/promptsource/seqio_tasks/tasks.py